scripts/flexible-server/auto-shutdown.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+#set -x
+shopt -s nocasematch
+AMBER='\033[1;33m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+SUBSCRIPTIONS=$(az account list -o json)
+jq -c '.[]' <<< $SUBSCRIPTIONS | while read subcription 
+do
+    SUBSCRIPTION_ID=$(jq -r '.id' <<< $subcription) 
+    SUBSCRIPTION_NAME=$(jq -r '.name' <<< $subcription) 
+    az account set -s $SUBSCRIPTION_ID
+    APPGS=$(az resource list --resource-type Microsoft.DBforPostgreSQL/flexibleServers --query "[?tags.autoShutdown == 'true']" -o json)
+   
+    jq -c '.[]'<<< $APPGS | while read app
+    do
+        SKIP="false"
+        app_id=$(jq -r '.id' <<< $app)
+        name=$(jq -r '.name' <<< $app)
+        rg=$(jq -r '.resourceGroup' <<< $app)
+        app_env=$(echo $SUBSCRIPTION_NAME | awk -F "-" '{ print $(NF) }')
+        app_env=${app_env/stg/Staging}
+        app_env=${app_env/sbox/Sandbox} 
+	app_env=${app_env/SHAREDSERVICESPTL/PTL}
+	if [[ $app_env =~ "INTSVC" ]] ; then
+	  temp_app_env=$(echo $SUBSCRIPTION_NAME | awk -F "-" '{ print $(NF-1) }')
```

Comment:
you have mixed tabs and spaces, can you format the file with something like `shfmt` please?
https://github.com/mvdan/sh